memory/stressng_mem.py.data/README

```diff
@@ -0,0 +1,51 @@
+ Description:
+-------------
+    - The main purpose of this test aims at stressing the memory component of the target system, using the memory stressors through the stress-ng program.
+    - The script also imposes a load on the CPU, but that's just a side-effect.
+    - The script uses a total of 28 stressors out of which we have 22 constant runtime stressors and 6 variable runtime stressors.
+
+ Input Parameters:
+------------------
+    --base_time    : 
+        - Specifies the time (in seconds) to run each stressor
+        - Deafult is 300 seconds or 5 minutes.
+
+    --time_per_gig :
+        - This specifies the time (in seconds) given to SOME stressors per GB
+        - Default is 60 seconds or 1 minute.
+        - 10 seconds per stressor, a total of 6 stressors hence 10 x 6 = 60s per GiB
+ 
+ How is the estimated execution time calculated?
+-------------------------------------------------
+    - Considering the defaults the Calculations are as below : 
+    
+    A] : For parameter --base-time       :   [ 28 x 300 ] 
+                                            = [8400 seconds] OR [140 minutes]
+    
+    B] : For parameter --time-per-gig    :   ([16gb x 60s] / 60)
+                                            = 16 minutes
+
+                                            OR
+
+                                            ([32gb x 60s] / 60)
+                                            = 32 minutes
+
+    Total Time taken = [A + B]
+                     = [140 + 16]
+                     = [156] minutes for a 16gb machine
+
+                     OR
+
+                     = [140 + 32]
+                     = [172] minutes for a 32gb machine
+
+ Method descriptions:
+----------------------
+    def execute_stressor():
+    - This method runs individual stressors - one at a time.
+    - Inputs : stressor, runtime
+    - Output : The variable "had_error" is set to value 1 in case of an error.
+
+    def calculate_variable_time()
+    - This method will estimate the time required for running the contant runtime stressors & the variable runtime stressors.
+    - The calculations required for these estimates are explained above in the "How is the estimated execution time calculated?" section.
```

Comment:
please remove this whitesace

Reply:
I do not see any whitespace at the end. Can you please point me exactly? I guess the remark say - "No newline at the end of file."